webpack.config.js

```diff
@@ -25,13 +25,18 @@ const config = {
     "ffi-napi": "commonjs ffi-napi",
   },
   resolve: {
-    extensions: ['.ts', '.js'],
-    extensionAlias: { '.js': ['.js', '.ts'] },
+    conditionNames: ["require", "node", "ts-loader"],

```

Comment:
Is webpack/ts-loader unable to find things using normal conditions? I guess I haven't seen a config which uses a `ts-loader` condition.

Reply:
I ran into an issue where Jest needed to find `out/path/to/file.js` at runtime, but webpack/ts-loader failed to resolve `out/path/to/file.js` back to `src/path/to/file.ts`. I added the condition so I could tell webpack/ts-loader where to find the source file.